.gitlintmodules

```diff
@@ -0,0 +1 @@
+wallet/desktop|jenkins
```

Comment:
jenkins should be allowed by default now, so not needed
cc: @Wayonb

Reply:
Yes, I can see, I removed => https://github.com/symbol/symbol/blob/b42863978fe9d32ed92922bd63c739a176fcd87f/linters/scripts/lint_last_commit.sh#L5